src/encoded/audit/experiment.py

```diff
@@ -2483,8 +2499,24 @@ def audit_experiment_target(value, system, excluded_types):
                                     'CRISPR genome editing followed by RNA-seq']:
         return
 
-    # Check that target of experiment matches target of antibody
     for rep in value['replicates']:
+        # Check target of experiment matches target of genetic modifications
+        if 'library' in rep and 'biosample' in rep['library']:
+            biosample = rep['library']['biosample']
+            for modification in biosample.get('applied_modifications', []):
```

Comment:
modification is a dict so we shouldn't default to list

Reply:
Again, "applied_modifications" is a list of dict.

Reply:
yes, reading that wrong